styles/helpers/_layout.scss

```diff
@@ -7,27 +7,27 @@ $subProps: ('', 'left', 'right', 'top', 'bottom');
 
 @include generate-helper($props, $subProps) using ($selector, $property, $style) {
   #{$selector}-double {
-    #{$property}: $layout-spacing-base * 2;
+    #{$property}: $layout-spacing-base * 2 !important;
```

Comment:
This makes me sad ☹️ but I've had troubles with these getting overridden too

Reply:
I thought this at first, but I think it will be ok!

Setting `!important` on something like divs globally would cause an issue, but as this class is added at a component level, it can always be removed!